src/constants.ts

```diff
@@ -1,2 +1,6 @@
 export const FIFTEEN_MINUTES_IN_MS = 1000 * 60 * 15
 export const ALFAJORES_CHAIN_ID = 44787
+
+export const VALORA_BASE_URL =
+  'https://liquidity-dot-celo-mobile-alfajores.appspot.com'
+export const VALORA_WEBHOOK_HISTORY_ENDPOINT = '/fiatconnect/webhook/history'
```

Comment:
these could be renamed `WEBHOOK_RECIPIENT_*` instead of "Valora"

Reply:
to avoid the implication that Valora is the only one that *can* receive webhooks